### ISSUE TITLE
Install using brew fails on Ubuntu 22.10

### ISSUE THREAD

**Issue by litearc** [clangen/musikcube#616]
I get an error stating that macOS is required when trying to install musikcube on Ubuntu 22.10 using brew:
```
brew install musikcube
musikcube: macOS is required for this software.
Error: musikcube: An unsatisfied requirement failed this build.
```
I was able to install the latest 2.x build I think.

**Comment by clangen** [comment]
Yeah, I simply haven't had the time to look into supporting brew on Linux. It didn't seem to work out of the box when I was creating the formula for macOS, so I explicitly disabled it. There's no reason it _shouldn't_ work, it probably just requires some tweaking.

**Comment by litearc** [comment]
Ok, I'll keep the ticket open then. The .deb package works fine, so it's not a major problem.

Thanks for all the hard work. This is a really awesome program!
